src/toil/batchSystems/mesos/batchSystem.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, config, maxCores, maxMemory, maxDisk, masterAddress):
         self.jobQueues = JobQueue()
 
         # Address of the Mesos master in the form host:port where host can be an IP or a hostname
-        self.masterAddress = masterAddress
+        self.masterAddress = config.masterAddress
```

Comment:
Name of destination in config is inconsistent. Here it is `masterAddress`. [Here](https://github.com/BD2KGenomics/toil/pull/1392/files#diff-b601d50a6eca36eb78f4bc23a332e93cR33) `mesosMasterAddress` is used.

[Here](https://github.com/BD2KGenomics/toil/pull/1392/files#diff-b601d50a6eca36eb78f4bc23a332e93cR84) `masterAddress` is also used.

I'm pretty sure that changing this to 
```
self.masterAddress = config.mesosMasterAddress
```
will solve at least some of the problems.